### ISSUE TITLE
Are there post Persona plans ?

### ISSUE THREAD

**Issue by fredhampton** [fedwiki/wiki#84]
I'am  interested in deploying a large wiki farm.
Are there currently alternatives to Persona ?
I m hesitant to begin a wiki farm without an understanding of what the direction will be post persona 

Thanks

**Comment by WardCunningham** [comment]
@paul90 is leading our work on modular security with plugins for the most obvious choices. See for example, https://github.com/fedwiki/wiki-security-passportjs.

**Comment by paul90** [comment]
The initial replacement for Persona is to provide authentication using a small set of [Passport](http://passportjs.org) strategies, currently GitHub, Google, and Twitter. Adding other strategies should be relatively simple.

Currently there is a wiki `next` branch, installable using `npm install wiki@next`, that includes this new security plug-in together with the associated changes to both the client and server. Documentation on configuration is currently in progress.

The longer term vision is to have an option that does not rely on an external identity provider - in all probability using webid-tls, webid-rsa or HTTP-Signature. But there were, and still are, a number of unanswered questions about those when started on this journey.

**Comment by herbsmn** [comment]
has anyone considered using [Indie Auth https://indieweb.org/indieauth](url) for this?

**Comment by WardCunningham** [comment]
Yes, I attended a presentation on Indie Auth at Indie Web Camp but that was before we had Paul's pluggable security infrastructure. I just remember being lost before the session was over.

**Comment by paul90** [comment]
I briefly looked at it, and IndieCert, for a replacement for Mozilla Persona. Both were dismissed as the assumptions they make about users were not valid for at least a portion of the existing wiki users.

I'm not aware of anybody currently considering a security plug-in using IndieAuth.
